database/changesets/20210422_TerraDataRepoSourceTable.xml

```diff
@@ -15,15 +15,27 @@
         <comment>
             the 'Terra DataRepo' source table schema
         </comment>
-        <sql>
-            CREATE TYPE TerraDataRepoSource AS (
-                id              bigint,      -- unique to this table
-                dataset         text,        -- the id of the TDR dataset
-                dataset_table   text,        -- the name of the dataset table
-                snapshot        text,        -- name of a column in table containing orderable and strictly increasing data
-                details         text,        -- the identifier of the source details table
-                last_checked    timestamptz  -- a timestamp used for monitor
-            )
-        </sql>
+        <createTable tableName="TerraDataRepoSource">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="dataset" type="text"
+                    remarks="the serialized EDN of the TDR dataset schema">
+                <constraints nullable="false"/>
+            </column>
+            <column name="dataset_table" type="text"
+                    remarks="the name of the dataset table">
+                <constraints nullable="false"/>
+            </column>
+            <column name="table_column_name" type="text"
+                    remarks="name of a column in table containing orderable and strictly increasing data">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_checked" type="timestamptz"
+                    defaultValueComputed="now()"
+                    remarks="a timestamp used for WFL to monitor new data on a rolling basis">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Does `last_checked` change?
What changes it?
Should the source of the column be consistent?

Reply:
This is meant to be the last time we checked the datarepo for new rows.

Reply:
I think this gets changed whenever the function that queries TDR finds new data. I'm not sure if `defaultValueComputed="now()"` is the right thing to do here thought. 

Also what do you mean by `Should the source of the column be consistent?`, could you elaborate?

Reply:
I meant that we shouldn't let the database decide on now the first time, and the server the rest of the time.
The server should set the time always or never.
(Same for the database, but it appears we want the server to manage it.)